### PR TITLE
:seedling: move estroz and adirio for emeritus (no longer directly involved)

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
move estroz and adirio for emeritus (no longer directly involved)

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@joelanford @jmrodri @rashmigottipati 

Both are no longer directly involved with the projects should we not move them for emeritus?

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -5,12 +5,10 @@ aliases:
   # tasks on the repo, or otherwise approve any PRs.
   kubebuilder-admins:
     - pwittrock
-    - estroz
     - camilamacedo86
 
   # non-admin folks who can approve any PRs in the repo
   kubebuilder-approvers:
-    - adirio
```
Comment (erikgb) [inline OWNERS_ALIASES:13]: Should this be an empty list to be valid? Or is it ok like this?

**Review by erikgb — COMMENTED** [review]
Maybe a little more redundancy is needed? The list of people with elevated permissions is short now...

File: OWNERS_ALIASES (hunk 1 above)
Comment (FillZpp) [reply OWNERS_ALIASES:13]: It seems ok to have empty list.

Comment (camilamacedo86) [reply OWNERS_ALIASES:13]: We need more contributions @erikgb and @FillZpp by sure
I hope that up to the end of this year we will be able to grow this list
and you both are very welcome as well ;-)

**Pull request merged**